examples/validation-and-warnings-codelab/complete-code/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  
+<head>
+  <meta charset="utf-8">
+  <title>Validation and Warnings Codelab</title>
+  <script src="https://unpkg.com/blockly/blockly.min.js"></script>
+  <script src="https://unpkg.com/@blockly/dev-tools@2.0.0/dist/index.js"></script>
+  <script src="index.js"></script>
```

Comment:
Are the dev-tools actually being used here? I don't see where. If not, remove them.

If they are, v2.0.0 is out of date; we are on v4 now. but since `blockly` is not versioned here, in the future this could get into a state where the latest version of blockly is not compatible with the exact version of dev-tools listed here. so they should either both be versioned or neither of them (since hopefully the latest version of dev-tools is always compatible with the latest version of blockly).

Reply:
Done, I don't think it's used here.

FYI, the custom toolbox codelab, which I copied this from, has the same outdated version and potential version compatibility problem, see: https://github.com/google/blockly-samples/blob/master/examples/custom-toolbox-codelab/starter-code/index.html